### PR TITLE
Docs only changes

### PR DESCRIPTION
**Pull request by gilescope**
Made it a bit clearer that tags are exclusive + fixed a few typos.

### CODE REVIEW

File: primitives/transaction-storage-proof/src/lib.rs [hunk 1]
```diff
@@ -108,26 +110,32 @@ impl sp_inherents::InherentDataProvider for InherentDataProvider {
...
+	/// including renewed transactions.
+	///
+	/// Note that this will only fetch transactions
+	/// that are indexed by the runtime with `storage_index_transaction`.
 	fn block_indexed_body(
 		&self,
 		number: NumberFor<B>,
 	) -> Result<Option<Vec<Vec<u8>>>, Error>;
 
+	/// Get block number by hash. Returns `None` if the header is not in the chain.
```
Comment (bkchr) [inline primitives/transaction-storage-proof/src/lib.rs:138]:
```suggestion
	/// Get block number for a block hash.
```

File: primitives/transaction-storage-proof/README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
-Authorship Primitives
+Transaction Storage Proof Primitives
+
+Constains types and basic code to extract storage proofs for indexed transactions.
+
+Why would you want to retrieve a transaction by its index?
```
Comment (bkchr) [inline primitives/transaction-storage-proof/README.md:5]:
```suggestion
```

File: primitives/transaction-storage-proof/README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
-Authorship Primitives
+Transaction Storage Proof Primitives
+
+Constains types and basic code to extract storage proofs for indexed transactions.
```
Comment (bkchr) [inline primitives/transaction-storage-proof/README.md:3]:
```suggestion
Contains types and basic code to extract storage proofs for indexed transactions.
```

File: client/transaction-pool/graph/src/ready.rs [hunk 4]
```diff
@@ -108,11 +108,14 @@ Hence every hash retrieved from `provided_tags` is always present in `ready`;
 qed
 "#;
 
+/// Validated transactions that are block ready with all their dependencies
+/// met.
```
Comment (bkchr) [inline client/transaction-pool/graph/src/ready.rs:112]:
```suggestion
/// Validated transactions that are block ready with all their dependencies met.
```

File: client/transaction-pool/graph/src/ready.rs [hunk 5]
```diff
@@ -235,7 +238,7 @@ impl<Hash: hash::Hash + Member + Serialize, Ex> ReadyTransactions<Hash, Ex> {
 			.fold(None, f)
 	}
 
-	/// Returns true if given hash is part of the queue.
+	/// Returns true if given transaction hash is part of the queue.
```
Comment (bkchr) [inline client/transaction-pool/graph/src/ready.rs:241]:
```suggestion
	/// Returns true if given transaction is part of the queue.
```

**Review by bkchr — APPROVED** [review]

**Review by thiolliere — APPROVED** [review]
seems good to me but I'm not familiar with the code, especially IndexedBody.

File: client/transaction-pool/graph/src/ready.rs [hunk 6]
```diff
@@ -108,11 +108,14 @@ Hence every hash retrieved from `provided_tags` is always present in `ready`;
...
 
+/// Validated transactions that are block ready with all their dependencies
+/// met.
 #[derive(Debug, parity_util_mem::MallocSizeOf)]
 pub struct ReadyTransactions<Hash: hash::Hash + Eq, Ex> {
-	/// Insertion id
+	/// Next free insertion id (used to indicate when a transaction was inserted into the pool).
 	insertion_id: u64,
 	/// tags that are provided by Ready transactions
+	/// (only one transaction can provide a tag)
```
[the remark this answers is not in the document]
Comment (gilescope) [reply client/transaction-pool/graph/src/ready.rs:117]:
```suggestion
	/// (only a single transaction can provide a specific tag)
```

**Pull request merged**